australian-guide-to-legal-citation.csl

```diff
@@ -199,11 +196,10 @@
         <text variable="title" font-style="italic" text-case="title" form="short"/>
       </if>
       <else>
-        <text variable="title" quotes="true" text-case="title" form="short"/>
+        <text variable="title" form="short" text-case="title" quotes="false"/>
```

Comment:
I don't think that's right. E.g. short forms of article titles should still be in quotes. You'll need a separate else-if here.